modules/commands/list.json

```diff
@@ -345,7 +363,7 @@
     "fields": [
       {
         "key": "Additional downloads",
-        "value": "https://ci.lucko.me/view/LuckPerms/"
+        "value": "https://ci.lucko.me/job/LuckPerms/"

```

Comment:
Linking to `view/` is correct here I think. luckperms.net already has the main downloads, "additional" should include everything else available on Jenkins which is LP related.